Buildings/Buildings.csv

```diff
@@ -1,298 +1,298 @@
-4611,L11,CLN,Columbia Lake Village North Block 11,,43.472568,-80.562495,1,43.47262,-80.56248
-4612,L12,CLN,Columbia Lake Village North Block 12,,43.472251,-80.56213,1,43.47231,-80.56216
-4613,L13,CLN,Columbia Lake Village North Block 13,,43.472885,-80.562699,1,43.47292,-80.5627
-4614,L14,CLN,Columbia Lake Village North Block 14,,43.472447,-80.562964,1,43.47249,-80.56295
-4615,L15,CLN,Columbia Lake Village North Block 15,,43.472148,-80.563334,1,43.47222,-80.56329
-4616,L16,CLN,Columbia Lake Village North Block 16,,43.472081,-80.56405,1,43.47211,-80.56404
-4617,L17,CLN,Columbia Lake Village North Block 17,,43.472451,-80.56364,1,43.4725,-80.56363
-4618,L18,CLN,Columbia Lake Village North Block 18,,43.472652,-80.563315,1,43.47269,-80.56331
-4619,L19,CLN,Columbia Lake Village North Block 19,,43.473284,-80.563176,1,43.47334,-80.56316
-4620,L20,CLN,Columbia Lake Village North Block 20,,43.473,-80.56349,1,43.47303,-80.56351
-4621,L21,CLN,Columbia Lake Village North Block 21,,43.472673,-80.563852,1,43.4727,-80.56387
-4622,L22,CLN,Columbia Lake Village North Block 22,,43.472356,-80.564235,1,43.47239,-80.56421
-4623,L23,CLN,Columbia Lake Village North Block 23,,43.471898,-80.564713,1,43.47199,-80.56467
-4624,L24,CLN,Columbia Lake Village North Block 24,,43.471659,-80.565011,1,43.4717,-80.56499
-4625,L25,CLN,Columbia Lake Village North Block 25,,43.471377,-80.565327,1,43.47145,-80.56534
-4626,L26,CLN,Columbia Lake Village North Block 26,,43.471194,-80.564995,1,43.47124,-80.565
-4627,L27,CLN,Columbia Lake Village North Block 27,,43.471153,-80.564517,1,43.4712,-80.56449
-4628,L28,CLN,Columbia Lake Village North Block 28,,43.471268,-80.56405,1,43.47132,-80.56402
-4629,L29,CLN,Columbia Lake Village North Block 29,,43.471359,-80.563578,1,43.4714,-80.56352
-4630,L30,CLN,Columbia Lake Village North Block 30,,43.471414,-80.562943,1,43.47146,-80.56295
-4631,L31,CLN,Columbia Lake Village North Block 31,,43.471719,-80.562948,1,43.47177,-80.56296
-4632,L32,CLN,Columbia Lake Village North Block 32,,43.471793,-80.56345,1,43.47184,-80.56344
-4633,L33,CLN,Columbia Lake Village North Block 33,,43.471606,-80.563895,1,43.47165,-80.56388
-4634,L34,CLN,Columbia Lake Village North Block 34,,43.471439,-80.564705,1,43.47148,-80.56471
-4635,L35,CLN,Columbia Lake Village North Block 35,,43.47185,-80.564235,1,43.47191,-80.56425
-A,SCS,,South Campus - South Grounds,,,,1,,
-B,SCN,,South Campus - North Grounds,,,,1,,
-C,,,Seagram Stadium Grounds,,,,0,,
-CGB,CGB,,Campus General Buildings,,,,1,,
-D,VLG,,Village 1 & 2 Grounds,,,,1,,
-E,MHG,,Minota Hagey Grounds,,,,1,,
-F,FCG,,Faculty Club Grounds,,,,1,,
-G,GAG,,Garage & Garbage,,,,1,,
-H,MSG,,Married Student Grounds,,,,1,,
-J,GRH,,Greenhouse,,,,1,,
-K,NCG,,North Campus Grounds,,,,1,,
-L,BAG,,Bauer Grounds,,,,1,,
-M,LL,,Laurel Lake,,,,1,,
-SCG,SCG,,South Campus General,,,,1,,
+building_id,building_code,parent_code,building_name,alternate_building_names,youtube_vid,streamable_vid,latitude,longitude,current,latitude_osm,longitude_osm
```

Comment:
@KartikTalwar I think we should append the new data to the right because folks might be doing some loosey goosey work with pulling in csv data. Thoughts?